.vscode/settings.json

```diff
@@ -4,4 +4,34 @@
     // Default (format when you save)
     "editor.formatOnSave": true,
     "editor.fontWeight": "normal",
+    "python.analysis.inlayHints.functionReturnTypes": false,
```

Comment:
I will include the prettier config for future PRs, could be useful for collaborators